ast_utils/string_buffer.c2

```diff
@@ -126,8 +128,8 @@ public fn void Buf.clear(Buf* buf) {
     buf.truncated = false;
 }
 
-public fn void Buf.color(Buf* buf, const char* color) {
-    if (buf.colors) buf.add(color);
+public fn void Buf.color(Buf* buf, Color col) {
+    if (buf.colors) buf.add(col.str());
```

Comment:
Inside col.str(), there is another check for if colors. Maybe have a separate str() function for use outside StringBuffer?

Reply:
Fixed: `col.str()` always returns the string associated with `col`.